.github/workflows/dockerimage.yml

```diff
@@ -17,6 +17,9 @@ jobs:
   build:
     runs-on: ubuntu-22.04
     steps:
+    - name: Delete tools folder
+      run: rm -rf /opt/hostedtoolcache
```

Comment:
Is this pre-populated on GitHub runners by default? This lacks documentation about what this is and why it needs to be removed

Reply:
Added docs